skrub/tests/test_datetime_encoder.py

```diff
@@ -143,7 +165,9 @@ def test_fit(
     add_day_of_the_week,
     resolution,
 ):
-    X = get_data_func(as_array=as_array)
+    if px is pl:
+        pytest.xfail(reason="Polars is not supported yet")
```

Comment:
This is solved creating pd.DataFrames first, then using `df = px.DataFrame(df)`. Now, all tests except for `test_mixed_type_dataframe` pass, and this one fails because of `X = to_datetime(X)` raises an error in Polars.

I assume it wasn't working when creating `px.DataFrames` directly because:

```
pl.DataFrame([
        ["2020-01-01", "2020-01-02", "2020-01-03"],
        ["2021-02-03", "2020-02-04", "2021-02-05"],
        ["2022-01-01", "2020-12-25", "2022-01-03"],
        ["2023-02-03", "2020-02-04", "2023-02-05"],
    ],
)
```
Is a 3 by 4 dataframe but

```
df = pd.DataFrame(
    [
        ["2020-01-01", "2020-01-02", "2020-01-03"],
        ["2021-02-03", "2020-02-04", "2021-02-05"],
        ["2022-01-01", "2020-12-25", "2022-01-03"],
        ["2023-02-03", "2020-02-04", "2023-02-05"],
    ],
)

pl.DataFrame(df)
```

is a 4 by 3 dataframe